torch/csrc/utils/tensor_numpy.cpp

```diff
@@ -136,8 +136,8 @@ at::Tensor tensor_from_numpy(PyObject* obj, bool warn_if_not_writeable/*=true*/)
   if (!PyArray_ISWRITEABLE(array) && warn_if_not_writeable) {
     TORCH_WARN_ONCE(
       "The given NumPy array is not writeable, and PyTorch does "
```

Comment:
Let's fix this comment slightly while we're here. Would you change all three instances of "writeable" to "writable"?

Reply:
Fixed